code/game/objects/items/storage/briefcase.dm

```diff
@@ -35,6 +35,24 @@
 	new /obj/item/stamp/law(src)
 	..()
 
+/obj/item/storage/briefcase/suicide_act(mob/user)
+	var/list/papers_found = list()
+	var/turf/T = get_turf(src)
+	for(var/atom/A in contents)
+		if(istype(A, /obj/item/paper) || istype(A, /obj/item/paperplane))
+			papers_found += A
+	if(!LAZYLEN(papers_found) || !T)
+		user.visible_message(span_suicide("[user] bashes [user.p_them()]self in the head with [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+		return BRUTELOSS
+
+	user.visible_message(span_suicide("[user] opens [src] and all of [user.p_their()] papers fly out!"))
+	for(var/obj/item/I in papers_found)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names